vanessaspear/passport-client
# Client can now see a list of their trips

## Description

When the client clicks on the trip tab, the app will show them a list of their trips.  They can access more information about the trip by clicking on the trip details button which will route them to the trip details page. 

## Changes
- Added a trip manager component to handle all fetch calls to the /trips resource
- Added a trips component to render the list of trips
- Added a trip component to generate the individual trip cards for the trip list
- Added a trip details components to link to the details button in the trip card (note: it doesn't have functionality to display the trip details yet)

## Steps to test

```
git fetch origin vs-trip-manager
git checkout vs-trip-manager 
```
- Login and click on Trips tab.  Verify the list of trips is displayed.  
```username=dgarcia@travel.com password: globetrotter```
- Click on the details button and verify the page is routed to http://localhost:3000/trips/{tripId}

## Related Issue
- Closes #2 
- Closes #3
